langchain4j-open-ai/src/main/java/dev/langchain4j/model/openai/OpenAiStreamingChatModel.java

```diff
@@ -208,67 +206,67 @@ private void generate(List<ChatMessage> messages,
         AtomicReference<String> responseModel = new AtomicReference<>();
 
         client.chatCompletion(request)
-                .onPartialResponse(partialResponse -> {
-                    responseBuilder.append(partialResponse);
-                    handle(partialResponse, handler);
-
-                    if (!isNullOrBlank(partialResponse.id())) {
-                        responseId.set(partialResponse.id());
+            .onPartialResponse(partialResponse -> {
```

Comment:
Would it be possible to configure formatting to avoid these big reformatting changes?

Reply:
@langchain4j, sure: https://github.com/langchain4j/langchain4j/pull/2082